### PR TITLE
added august talks and mc slides

### PR DESCRIPTION
**Pull request by weizhang9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ Our events follow the Go Community [Code of Conduct](https://golang.org/conduct)
 
 * We help refine your topic and presentation
 * Register your interest via [this short form](https://goo.gl/forms/1VAAejXiA69bXXCP2)!
+* Please feel free to refer to MC's slides for topic ideas: [MC slides 2019](https://drive.google.com/drive/folders/19r-T_rNVIAxZz0i54vxd6a_nQAixiqj8?usp=sharing) and [MC slides 2018](https://drive.google.com/drive/folders/1ALIWCIzv6NFJbJ22rU7FP5Tiu8lrgy-F?usp=sharing)
```
Comment (mvdan) [inline README.md]: I'd add this under Records, as it's useful for more than just new speakers.

Also, I'd share the entire MC slides directory, which contains every year as a sub-folder.

Comment (weizhang9) [reply README.md]: the folder also has sponsor's slides, is that ok to share?

Comment (mvdan) [reply README.md]: Ah, we can fix that. I'll move the sponsor slides elsewhere in a sec.

Comment (mvdan) [reply README.md]: Done.

Comment (weizhang9) [reply README.md]: thank you, i have made the changes now :)

**Pull request merged**